packages/devtools/lib/src/debugger/debugger_state.dart

```diff
@@ -20,32 +21,34 @@ class DebuggerState {
 
   final Map<String, Script> _scriptCache = <String, Script>{};
 
-  final BehaviorSubject<bool> _paused = BehaviorSubject<bool>.seeded(false);
-  final BehaviorSubject<bool> _supportsStepping =
-      BehaviorSubject<bool>.seeded(false);
+  final _isPaused = ValueNotifier<bool>(false);
+  ValueListenable<bool> get isPaused => _isPaused;
+
+  final _hasFrames = ValueNotifier<bool>(false);
+  ValueNotifier<bool> _supportsStepping;
+  ValueListenable<bool> get supportsSteppingListenable =>
```

Comment:
nit: maybe use `{ }` instead of `=>` for this getter as dartfmt indent is a bit odd for this case.

Reply:
It doesn't make a huge difference, 2 fewer spaces of indent helps a little I suppose.